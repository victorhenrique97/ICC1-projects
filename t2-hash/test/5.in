487
500
494
653
814
267
674
319
444
214
753
863
300
929
687
627
961
726
228
125
123
137
933
523
532
134
171
543
614
165
277
991
983
271
135
715
328
427
977
784
859
287
508
963
725
566
178
553
867
250
796
312
301
305
368
908
595
702
512
935
816
673
675
773
252
648
601
835
868
927
304
626
747
635
307
150
678
847
688
119
692
217
136
972
152
734
401
798
207
894
842
620
434
170
447
349
986
241
453
776
179
839
699
190
211
915
269
763
624
713
491
641
248
159
819
660
603
416
337
782
741
716
361
195
448
925
120
372
461
831
261
324
930
309
121
258
208
450
652
871
527
791
962
224
757
499
138
596
995
617
923
936
573
128
876
325
296
984
477
474
685
644
393
890
219
139
589
365
404
799
701
182
905
443
817
452
636
779
804
164
952
227
826
809
764
283
667
492
607
938
355
966
709
668
268
957
545
976
340
205
338
661
931
472
766
896
462
634
557
807
710
441
437
754
274
886
829
546
854
700
236
838
756
420
563
397
240
805
292
369
333
571
913
391
162
294
895
201
567
424
943
558
778
889
650
230
411
921
582
951
177
950
956
580
247
473
221
774
649
140
646
484
431
380
909
196
525
934
583
872
928
344
672
907
131
898
278
259
225
953
893
906
373
345
577
562
969
332
215
947
212
903
363
275
992
744
320
730
561
398
127
780
822
729
769
238
276
860
735
738
180
803
574
117
295
883
102
877
958
706
456
743
144
308
916
156
602
768
937
487
981
504
613
229
633
550
203
579
611
910
658
104
115
698
714
618
654
410
132
400
298
534
901
569
281
493
761
116
657
865
419
845
861
578
911
918
810
681
967
126
1000
625
302
677
148
695
680
167
914
351
154
438
758
109
857
808
944
615
830
727
770
141
415
811
924
489
849
659
197
321
576
297
733
422
833
237
749
559
160
529
358
586
446
556
469
339
293
820
408
133
548
606
987
189
600
310
585
759
402
794
467
850
789
748
540
875
575
682
832
881
199
396
323
498
737
392
313
490
374
395
608
917
192
902
827
518
209
637
605
643
628
501
973
451
560
270
303
792
825
651
353
640
343
218
879
198
707
386
846
371
458
751
440
812
978
122
181
389
515
869
468
612
522
862
590
623
106
272
511
728
260
760
