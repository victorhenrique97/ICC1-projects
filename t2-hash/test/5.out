0: 487
1:
2: 976 489
3: 977 490
4: 491 978
5: 492
6: 493
7: 494 981
8:
9: 983
10: 984
11: 498
12: 986 499
13: 987
14: 501
15:
16:
17: 991 504
18: 992
19:
20:
21: 508 995
22:
23:
24: 511
25: 512
26: 1000
27:
28: 515
29:
30:
31: 518
32:
33:
34:
35: 522
36: 523
37:
38: 525
39:
40: 527
41:
42: 529
43:
44:
45: 532
46:
47: 534
48:
49:
50:
51:
52:
53: 540
54:
55:
56: 543
57:
58: 545
59: 546
60:
61: 548
62:
63: 550
64:
65:
66: 553
67:
68:
69: 556
70: 557
71: 558
72: 559
73: 560
74: 561
75: 562
76: 563
77:
78:
79: 566
80: 567
81:
82: 569
83:
84: 571
85:
86: 573
87: 574
88: 575
89: 576
90: 577
91: 578
92: 579
93: 580
94:
95: 582
96: 583
97:
98: 585
99: 586
100:
101:
102: 589 102
103: 590
104: 104
105:
106: 106
107:
108: 595
109: 596 109
110:
111:
112:
113: 600
114: 601
115: 602 115
116: 603 116
117: 117
118: 605
119: 119 606
120: 120 607
121: 121 608
122: 122
123: 123
124: 611
125: 125 612
126: 613 126
127: 614 127
128: 128 615
129:
130: 617
131: 131 618
132: 132
133: 620 133
134: 134
135: 135
136: 136 623
137: 137 624
138: 138 625
139: 626 139
140: 627 140
141: 141 628
142:
143:
144: 144
145:
146: 633
147: 634
148: 635 148
149: 636
150: 150 637
151:
152: 152
153: 640
154: 641 154
155:
156: 156 643
157: 644
158:
159: 159 646
160: 160
161: 648
162: 162 649
163: 650
164: 164 651
165: 165 652
166: 653
167: 654 167
168:
169:
170: 170 657
171: 171 658
172: 659
173: 660
174: 661
175:
176:
177: 177
178: 178
179: 179
180: 667 180
181: 668 181
182: 182
183:
184:
185: 672
186: 673
187: 674
188: 675
189: 189
190: 190 677
191: 678
192: 192
193: 680
194: 681
195: 195 682
196: 196
197: 197
198: 685 198
199: 199
200: 687
201: 688 201
202:
203: 203
204:
205: 692 205
206:
207: 207
208: 208 695
209: 209
210:
211: 211 698
212: 699 212
213: 700
214: 214 701
215: 702 215
216:
217: 217
218: 218
219: 219 706
220: 707
221: 221
222: 709
223: 710
224: 224
225: 225
226: 713
227: 227 714
228: 228 715
229: 716 229
230: 230
231:
232:
233:
234:
235:
236: 236
237: 237
238: 725 238
239: 726
240: 240 727
241: 241 728
242: 729
243: 730
244:
245:
246: 733
247: 734 247
248: 248 735
249:
250: 250 737
251: 738
252: 252
253:
254: 741
255:
256: 743
257: 744
258: 258
259: 259
260: 747 260
261: 261 748
262: 749
263:
264: 751
265:
266: 753
267: 267 754
268: 268
269: 269 756
270: 757 270
271: 271 758
272: 759 272
273: 760
274: 274 761
275: 275
276: 763 276
277: 277 764
278: 278
279: 766
280:
281: 768 281
282: 769
283: 283 770
284:
285:
286: 773
287: 287 774
288:
289: 776
290:
291: 778
292: 779 292
293: 780 293
294: 294
295: 782 295
296: 296
297: 784 297
298: 298
299:
300: 300
301: 301
302: 302 789
303: 303
304: 304 791
305: 305 792
306:
307: 307 794
308: 308
309: 796 309
310: 310
311: 798
312: 312 799
313: 313
314:
315:
316: 803
317: 804
318: 805
319: 319
320: 807 320
321: 808 321
322: 809
323: 810 323
324: 324 811
325: 325 812
326:
327: 814
328: 328
329: 816
330: 817
331:
332: 819 332
333: 333 820
334:
335: 822
336:
337: 337
338: 338 825
339: 826 339
340: 340 827
341:
342: 829
343: 830 343
344: 831 344
345: 345 832
346: 833
347:
348: 835
349: 349
350:
351: 838 351
352: 839
353: 353
354:
355: 842 355
356:
357:
358: 845 358
359: 846
360: 847
361: 361
362: 849
363: 363 850
364:
365: 365
366:
367: 854
368: 368
369: 369
370: 857
371: 371
372: 859 372
373: 373 860
374: 861 374
375: 862
376: 863
377:
378: 865
379:
380: 867 380
381: 868
382: 869
383:
384: 871
385: 872
386: 386
387:
388: 875
389: 876 389
390: 877
391: 391
392: 392 879
393: 393
394: 881
395: 395
396: 883 396
397: 397
398: 398
399: 886
400: 400
401: 401
402: 889 402
403: 890
404: 404
405:
406: 893
407: 894
408: 895 408
409: 896
410: 410
411: 411 898
412:
413:
414: 901
415: 415 902
416: 416 903
417:
418: 905
419: 906 419
420: 420 907
421: 908
422: 909 422
423: 910
424: 424 911
425:
426: 913
427: 427 914
428: 915
429: 916
430: 917
431: 431 918
432:
433:
434: 434 921
435:
436: 923
437: 437 924
438: 925 438
439:
440: 927 440
441: 441 928
442: 929
443: 930 443
444: 444 931
445:
446: 933 446
447: 447 934
448: 935 448
449: 936
450: 450 937
451: 938 451
452: 452
453: 453
454:
455:
456: 943 456
457: 944
458: 458
459:
460: 947
461: 461
462: 462
463: 950
464: 951
465: 952
466: 953
467: 467
468: 468
469: 956 469
470: 957
471: 958
472: 472
473: 473
474: 961 474
475: 962
476: 963
477: 477
478:
479: 966
480: 967
481:
482: 969
483:
484: 484
485: 972
486: 973
